packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -132,8 +133,13 @@ export function setValueForProperty(
       const attributeName = name;
       if (value === null) {
         node.removeAttribute(attributeName);
+      } else if (__DEV__) {
+        node.setAttribute(
+          attributeName,
+          stringifyWithPerformanceWarning(value),
+        );
       } else {
-        node.setAttribute(attributeName, '' + (value: any));
+        node.setAttribute(attributeName, (value: any).toString());
```

Comment:
isn't this will throw on `undefined` value? we only strictly check for `null` above it.

Reply:
Yeah, @Swieckowski please revert this. We were intentionally not using `toString`